server/jobs/index.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable import/no-dynamic-require */
+/* eslint-disable global-require */
+import fs from 'fs';
+
+const getJobs = (folderName) => {
+  const jobs = fs
+    .readdirSync(`${__dirname}/${folderName}`)
+    .map((fileName) => {
+      const { default: exported } = require(`${__dirname}/${folderName}/${fileName}`);
```

Comment:
What if you just did:
`const { exported }= require(`${__dirname}/${folderName}/${fileName}`);`, will it not work?